src/components/main.css

```diff
@@ -0,0 +1,21 @@
+.navlinks {
```

Comment:
There is something quite subtle but that helps a lot in the communication as a team, it's agreeing on a code style, for example in this css file you have used both **CamelCase** notation and **lowercase** in the names, **AddItem**, **selected**, etc And for example @hajarNasr and @saraheisa have choosed lowercase with dashes ej: **add-item**. This comment is non blocking, you don't need to change anything, I only wanted to trigger some reflection

Reply:
Yes, good point! Thank you for pointing it out! 
I'll bring it up to the group so we can agree on a code style, specially for the CSS file 👍